### DISCUSSION TITLE
AMA

### DISCUSSION THREAD

**Discussion by yaocccc** [yaocccc/dwm#10] [Announcements]
## 👋 Welcome!

you can ask my anything

**Comment by fecet** [comment]
overview能否使用tag作为level，目前如果一个tag下打开了很多窗口，overview会全部显示出来，起不到overview的目的

**Reply by yaocccc** [reply]
是类似于把每个Tag都做成缩略图平铺成来，然后再去选某一个是吧，这个可有点难搞哦

**Reply by fecet** [reply]
对的, 确实感觉有点难搞..

**Reply by yaocccc** [reply]
对的，涉及到图片生成，你还要在界面上，把缩略图都合理的平铺出来，好家伙。

**Reply by fecet** [reply]
https://dwm.suckless.org/patches/tag-previews/
这个可能有用
